src/components/Select/Select.story.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
Use one `Select.stories.mdx` instead of this outdated mdx + jsx pattern (see Example component)

Reply:
ah I copied an old component. I'll update it while I'm here.